src/GitHub.InlineReviews/GitHub.InlineReviews.csproj

```diff
@@ -15,7 +15,7 @@
   <PropertyGroup>
     <SignAssembly>true</SignAssembly>
   </PropertyGroup>
-  <PropertyGroup>
+  <PropertyGroup Condition="$(Buildtype) == 'Internal'">
```

Comment:
Same as above, the check here should include `'` for safety

`Condition="'$(BuildType)' != 'Internal'"`